app/core/services.py

```diff
@@ -77,6 +78,15 @@ def get_service_object(
     else:
         service["downloaded"] = False
 
+    proxy_enabled = os.environ.get("PROXY_ENABLED", "false").lower() == "true"
```

Comment:
I would put `proxy_enabled` env variable loading with the rest of the configs: https://github.com/premAI-io/prem-daemon/blob/main/app/core/config.py

Reply:
Done

Reply:
Instead of strings I would use boolean.

```python
PROXY_ENABLED = os.getenv("PROXY_ENABLED", False)
```

and use it in services:

```python
if config.PROXY_ENABLED:
    do_stuff()
```

Reply:
Done

Reply:
I don't see the changes. Did you push the code? @sekulicd